src/sphinx/4.0.x/features/scaladebugger/index.rst

```diff
@@ -144,28 +134,56 @@ Using the Scala debugger for a remote debugging session is done by selecting one
 .. image:: images/remotedebugging-01.png
    :alt: Scala remote debugging configuration
 
-Status
-------
+Drop to frame |new| (since 4.0)
+...............................
 
-* Smart stepping over/in/out anonymous functions (for comprehension, collection methods, ...).
-* Resume/terminate launch action.
-* The structure of the Scala implementation of the Eclipse debug model.
-* Improved label and value display.
-* No good icons for the model elements.
-* No caching of any data.
-* No drop to frame.
-* Need to define and spec the exact behavior of the smart step in/over/out.
+Select the ``Drop to Frame`` command to re-enter the selected stack frame in the ``Debug`` View.
 
-known issues
-............
+.. image:: images/drop-to-frame.png
+   :alt: drop to frame option in menu
+
+.. note:: This command is only available if the current VM supports drop to frame and the selected stackframe is not in a native method.
+
+Hot code replace |new| (since 4.1)
+----------------------------------
+
+Hot code replace adds the possibility to modify and re-compile code in a debug mode and to have these changes visible
```

Comment:
Are these new lines in the middle of sentences needed (maybe some formatting requirements)? Or do we have them e.g. because they were in the description of a commit/PR?

I'm just asking. :)